ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiConfigManagementService.scala

```diff
@@ -110,7 +112,7 @@ private[apiserver] final class ApiConfigManagementService private (
           index,
           ledgerEndBeforeRequest,
         ),
-        timeToLive = params.timeToLive,
+        timeToLive = JDuration.ofMillis(params.timeToLive.toMillis),
```

Comment:
Why does our Scala class expect a Java duration?

Reply:
Because Scala durations are really only intended for concurrency. We probably shouldn't be using them in configuration either, but the string format is more pleasant than the Java duration (we can parse "30s" instead of "PT30S").

Reply:
I see. From a usability perspective then it sounds like we should pick the Scala durations.